src/freenet/clients/http/N2NTMToadlet.java

```diff
@@ -285,6 +289,14 @@ public static void createN2NTMSendForm(HTMLNode pageNode, boolean advancedMode,
 		}
 		messageForm.addChild("textarea", new String[] { "id", "name", "rows",
 				"cols" }, new String[] { "n2ntmtext", "message", "8", "74" });
+		if (fProxyJavascriptEnabled) {
+			HTMLNode letterCountInfo = messageForm.addChild("div", new String[] {"class"}, new String[] {"n2ntmcountouter"});
+			letterCountInfo.addChild("#", NodeL10n.getBase().getString("N2NTMToadlet.remainingLetters"));
+			letterCountInfo.addChild("span", new String[] {"id"}, new String[] {"n2ntmcount"});
+			letterCountInfo.addChild("script", new String[] {"type"}, new String[] {"text/javascript"})
```

Comment:
I’m calling `countletters` to pass the element IDs to the js code, so we can re-use this in different places or for multiple text-fields on one page.